### PR TITLE
Contact Management

### PR DESCRIPTION
**Pull request by steveoh**
refs: #611
This creates a `_data/contacts.yml` that has all the contacts in our data pages. 

We can reference contacts in pages with the `_includes/contact.html`
The include has a default text for what it puts before the contact information. You can update that text with the `text` attribute of the include. The `subject` property updates the `mailto` subject link. It is acceptable to use `page.title` unless you would like a more precise name. `hide-punctuation=true` will remove the ending `.` otherwise the include html is all jacked up and if you try to add your own punctuation is get's weird. 

Contacts can be referenced by `site.data.contacts.agrc`.

When using inside of markdown things get a little wonky. You have to create a capture group and then send the output of the capture group through the `strip_newlines` filter because the include html is all weird.

```liquid
{% capture liaison %}{% include contact.html subject=page.title contact=site.data.contacts.luca_liaison hide-punctuation=true text=' ' %}{% endcapture %}
```

otherwise you can do

```liquid
{% include contact.html subject=page.title contact=site.data.contacts.address_grids text='or email'  %}
```

@BGranberg will you please review the `/_data/contacts.yml` to make sure the people are current enough.

### CODE REVIEW

File: _data/contacts.yml [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  name: Gordon Douglass
+  email: GDouglass@utah.gov
+  phone: 801-538-4810
+  agency: UGS
+water_related_land_use:
+  name: Aaron Austin
+  email: AaronAustin@utah.gov
+  agency: DWR
+voting:
+  name: Mark Thomas
```
Comment (BGranberg) [inline _data/contacts.yml]: Justin Lee

File: _data/contacts.yml [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  email: MJThomas@utah.gov
```
Comment (BGranberg) [inline _data/contacts.yml]: justinlee@utah.gov

File: _data/contacts.yml [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+  agency: DNR-OGM
+oil_fields:
+  name: Brad Hill
+  emal: BradHill@utah.gov
+  agency: DNR-OGM
+air_quality:
+  name: Whitney Oswald
+  email: WOswald@utah.gov
+  agency: DAQ
+brownsfield:
```
Comment (BGranberg) [inline _data/contacts.yml]: brownfields

**Comment by steveoh** [comment]
merged

**Pull request closed without merging**